src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -77,6 +77,69 @@ internal static ParseResult ParseScript(string script, ILanguageSettings loc = n
             return new ParseResult(parsetree, errors, errors?.Any() ?? false, parser._comments, parser._before, parser._after);
         }
 
+        public static ParseFormulasResult ParseFormulasScript(string script, ILanguageSettings loc = null, Flags flags = Flags.None)
```

Comment:
A few questions:

1. Why can't we use the ParseInstance?
2. Is a=10;; a valid expression in this mini-language?
3. Why does this static function perform the task of parsing? The existing ParseScript static method does not do any parsing. It delegates to the .Parse() instance method.
4. Why are we doing such low-level manipulation of the instance variables of an object in a static method like this? Shouldn't we have an instance method?

Reply:
I refactored to make it an instance method.

Reply:
2. No